src/akamai/js/akamaiHD_flash.js

```diff
@@ -242,6 +243,9 @@ require("../../../html5-common/js/utils/constants.js");
      * @param {boolean} rewind True if the stream should be set to time 0
      */
     this.load = function(rewind) {
+      if (loaded && !rewind) return;
+      this.callToFlash("load("+rewind+")");
+      loaded = true;
```

Comment:
Forgot to mention that loaded is never set to false again.  It needs to be set to false when the stream URL changes